integration-tests/src/test/java/oracle/kubernetes/operator/ITSessionMigration.java

```diff
@@ -0,0 +1,393 @@
+    // bf and af primaryServName1 stopped
+    Assume.assumeTrue(
+        "HTTP session state is NOT migrated!", countattribute1.equals(countattribute2));
+
+    // restore test env
+    // TODO will use this when it works
+    // domain.restartManagedServerUsingServerStartPolicy(primaryServName1);
+    // wait sometime for ms pod gets ready
+    Thread.sleep(30 * 1000);
+    TestUtils.checkPodCreated(domainUid + "-" + primaryServName1, domainNS);
+    TestUtils.checkPodReady(domainUid + "-" + primaryServName1, domainNS);
+
+    logger.info("SUCCESS - " + testMethodName + ". HTTP session state is migrated!");
+  }
+
+  @Ignore
+  @Test
+  public void myTest1() throws Exception {
+    Assume.assumeFalse(QUICKTEST);
+    Map<String, Object> domainMap = domain.getDomainMap();
+    String domainNS = domainMap.get("namespace").toString();
+
+    domain.shutdownManagedServerUsingServerStartPolicy("managed-server2");
+    TestUtils.checkPodDeleted("domainonpvwlst-managed-server2", domainNS);
+  }
+
+  @Ignore
+  @Test
+  public void myTest2() throws Exception {
+    Assume.assumeFalse(QUICKTEST);
+    Map<String, Object> domainMap = domain.getDomainMap();
+    String domainNS = domainMap.get("namespace").toString();
+
+    domain.shutdownManagedServerUsingServerStartPolicy("domainonpvwlst-managed-server1");
+    TestUtils.checkPodDeleted("domainonpvwlst-managed-server1", domainNS);
+  }
+
+  /**
+   * build
+   *
+   * @throws Exception
+   */
+  private String getHttpResponseAttribute(String httpResponseString, String attribute)
+      throws Exception {
+    // logger.info("Looking for <" + attribute + "> in\n " + httpResponseString);
+    String attrPatn = httpAttrMap.get(attribute);
+
+    Assume.assumeNotNull(attrPatn);
+
+    String httpAttribute = null;
+
+    Pattern pattern = Pattern.compile(attrPatn);
+    Matcher matcher = pattern.matcher(httpResponseString);
+    if (matcher.find()) {
+      httpAttribute = matcher.group(2);
+    }
+
+    // logger.info("Value for HTTP attribute found.: " + attribute + " = " + httpAttribute);
+
+    return httpAttribute;
+  }
+
+  /**
+   * build
+   *
+   * @throws Exception
+   */
+  private String buildWebServiceUrl(String curlURLPath, String paramToAppend) throws Exception {
+    String nodePortHost = domain.getHostNameForCurl();
+    int nodePort = domain.getLoadBalancerWebPort();
+
+    StringBuffer webServiceUrl = new StringBuffer("curl --silent ");
+    webServiceUrl
+        .append(" -H 'host: ")
+        .append(domain.getDomainUid())
+        .append(".org' ")
+        .append(" http://")
+        .append(nodePortHost)
+        .append(":")
+        .append(nodePort)
+        .append("/")
+        .append(curlURLPath)
+        .append(paramToAppend);
+
+    logger.info("Built web service url: " + webServiceUrl);
+
+    return webServiceUrl.toString();
+  }
+
+  /**
+   * build
+   *
+   * @throws Exception
+   */
+  private ExecResult execCurlCmd(String curlCmd) throws Exception {
```

Comment:
See if you can use TestUtils.exec() method.

Reply:
Thanks. The change is on the way